packages/fxa-content-server/app/scripts/views/sms_send.js

```diff
@@ -142,6 +143,29 @@ class SmsSendView extends FormView {
     );
   }
 
+  /**
+   * Check if the current user is already signed in.
+   *
+   * @returns {Boolean}
+   * @private
+   */
+  _isSignedIn() {
```

Comment:
This method is present in [ConnectAnotherDeviceView](https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/app/scripts/views/connect_another_device.js#L179-L194) so I wanted to also bring over the comment context.